csrc/host_ir/host_ir.h

```diff
@@ -173,7 +173,7 @@ class Deallocate : public Expr {
     return "hir::Deallocate";
   }
 
-  const kir::Allocate* allocation() const;
+  TensorView* allocation() const;
```

Comment:
Rename to `buffer`? English-wise, `allocation` is an action to allocate a buffer but not the buffer. So it can be confusing.